core-bundle/src/Command/CrawlCommand.php

```diff
@@ -75,6 +75,7 @@ protected function configure(): void
             ->addOption('no-progress', null, InputOption::VALUE_NONE, 'Disables the progress bar output')
             ->addOption('enable-debug-csv', null, InputOption::VALUE_NONE, 'Writes the crawl debug log into a separate CSV file')
             ->addOption('debug-csv-path', null, InputOption::VALUE_REQUIRED, 'The path of the debug log CSV file', Path::join(getcwd(), 'crawl_debug_log.csv'))
+            ->addOption('ignore-certificate', null, InputOption::VALUE_NONE, 'Ignore the TLS/SSL certificate of indexed hosts')
```

Comment:
Naming examples from other tools:

- `curl`: `--insecure`
- Chrome:  `--ignore-certificate-errors`

But `ignore-certificate` seems valid to me as you set `verify_peer` and `verify_host` to `false` and therefore a certificate error cannot occur anymore.